script.js

```diff
@@ -303,3 +308,36 @@ function appendStory(story, board, storyUrl, statusBadge, estimate) {
     "</div>"
   );
 }
+
+function initializeDragAndDrop () {
+  $(".todo, .in-progress, .ready-for-review, .done").sortable({
+    connectWith: ".todo, .in-progress, .ready-for-review, .done"
+  });
+
+  $(".js-board").on("sortstop", function(event, ui) {
+      updatePivotal(ui);
+  });
+} //end initializeDragAndDrop
+function updatePivotal(ui) {
+    let id = ui.item.data("id");
+    console.log(ui.item);
+    let newStatus = ui.item.closest(".js-board").data("status");
+    let beforeId = ui.item.prev().data("id");
+    let urlId = "/" + ui.item.data("id");
+    if(beforeId === undefined) beforeId = null;
+    instance.put("/" + id, {
+    "current_state": newStatus,
+})
```

Comment:
The sort order within a list is not working. Looks like you're not passing the `before_id` at all? Or `after_id` if it's at the end of the list?

Reply:
opps! i only did it in my mind!

Reply:
What happens if it's first in the list?